packages/asset-swapper/src/utils/market_operation_utils/orders.ts

```diff
@@ -4,7 +4,7 @@ import { ERC20BridgeAssetData, SignedOrder } from '@0x/types';
 import { AbiEncoder, BigNumber } from '@0x/utils';
 
 import { MarketOperation, SignedOrderWithFillableAmounts } from '../../types';
-import { RfqtIndicativeQuoteResponse } from '../quote_requestor';
+import { IndicativeQuote } from '../quote_requestor';
```

Comment:
Would prefer to import this directly from `@0x/quote-server`

Reply:
My logic was that a client should import an interface's dependent types from the module that exports that interface.  Again, happy to defer to convention as you see fit, but this is what made sense to me.

Reply:
It feels more direct to import directly from `@0x/quote-server`, and I see us using this pattern elsewhere